### PR TITLE
Issue Of Readme Solved

### LINKED ISSUE #8: [Feature] Improve contribution section from readme
## Feature description

Improve the contribution section from the readme file:
- Add how to contribute
- Add contributors

Also add a CONTRIBUTING.md file with contribution guidelines.

### PR DESCRIPTION
**Pull request by cyberpvn7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,27 @@ export default function App() {
...
+ ```
+ 
+ ### Star The Repository :star2:
+
 ## Props
 * buttonName - string for button name
 * hoverBtnName - string for hover button name
 
 ## Contributing
-If you think you can make an improvement don't hesitate to open a pull request.
```
Comment (eduardconstantin) [inline README.md:31]: Don't remove this line

File: README.md [hunk 2]
```diff
@@ -23,9 +23,27 @@ export default function App() {
...
+## Awesome contributors :star_struck:
+<a href="https://github.com/eduardconstantin/Comic-book-button-anim/graphs/contributors">
+  <img src="https://contributors-img.web.app/image?repo=kishanrajput23/Awesome-Python-Projects" />
```
Comment (eduardconstantin) [inline README.md]: Please change contributors list

**Comment by cyberpvn7** [comment]
Hey Can You Add label of hacktoberfest?

**Review by cyberpvn7 — COMMENTED** [review]
Review Complete

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cyberpvn7) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (cyberpvn7) [reply README.md:31]: Done

**Comment by cyberpvn7** [comment]
Can you please  add this PR in **Hacktoberfest**

**Comment by eduardconstantin** [comment]
Yes, looks ok now. Thank you!

**Comment by cyberpvn7** [comment]
Your Welcome

**Pull request merged**